File: lua/fizzbuzz.lua
local function fizzbuzz(n)
	for i=1, n do
		if i % 3 == 0 and i % 5 == 0 then
			print("FizzBuzz")
		end
		if i % 3 == 0 then
			print("Fizz")
		end
		if i % 5 == 0 then
			print("Buzz")
		end
		print(i)
	end
end

fizzbuzz(100)

File: go/src/fizzbuzz.go
package main

import "fmt"

func main() {
	for i := 0; i <= 42; i += 2 {
		if (i%6 == 0) && (i%7 == 0) {
			fmt.Println("Fizz Buzz")
		} else if i%6 == 0 {
			fmt.Println("Fizz")
		} else if i%7 == 0 {
			fmt.Println("Buzz")
		} else {
			fmt.Println(i)
		}
	}
}
